.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.8"
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Prepare environment
+        env:
+          PRIVATE_REPOSITORY_SBT: ${{ secrets.PRIVATE_REPOSITORY_SBT }}
+          SONATYPE_FILE: ${{ secrets.SONATYPE_FILE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+        run: |
+          mkdir --parents ~/.sbt/1.0 || true
+          echo "$PRIVATE_REPOSITORY_SBT" > ~/.sbt/1.0/privateRepository.sbt
+          echo "$SONATYPE_FILE" > ~/.sbt/1.0/sonatype.sbt
+          echo "$PGP_SECRET" | base64 --decode | gpg --batch --import
+
+#     - name: Cache
+#       No cache for release! We want clean and full release!
```

Comment:
I would like some elements to support this decision. We had mentioned that javascript dependencies versions are often overwritten, but it will not be the case here?

Reply:
1. It happened that we did it.
2. We want to ensure dependencies are available (example: bintray sunset).
3. We don't want to have anything that can disturb the reproduction of the release.

Reply:
Thank you!